tests/models/packages/test_ebuild.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2021-2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+from pathlib import Path
+from unittest import TestCase
+
+
+from notus.scanner.models.packages.ebuild import EBuildPackage
+
+
+class EBuildPackageTestCase(TestCase):
+    def test_parse_fullname(self):
+        """
+        verifies if ebuild id able to parse the fullname examples provided by
+        gentoo_examples.txt
+        """
+        currentp = Path(__file__).parent
+        with open(
+            f"{currentp}/gentoo_examples.txt", "r", encoding="utf-8"
+        ) as file:
+            fullname = file.readline()
+            if not EBuildPackage.from_full_name(fullname):
+                self.fail(f"{fullname} is not parseable for EBuildPackage.")
+
```

Comment:
Test for correct parsing. Check if name and version is set correctly.

Reply:
That is tested in test_comparability.